test/test.js

```diff
@@ -0,0 +1,298 @@
+import { expect } from 'chai';
+import GeoServerRestClient from '../geoserver-rest-client.js';
+
+const url = 'http://localhost:8080/geoserver/rest/';
```

Comment:
I think this needs to be configurable by env var for CI (same for user / password)